crates/api/src/fetch.rs

```diff
@@ -34,17 +42,29 @@ pub fn serialize_op_ids(value: Vec<OpId>) -> bytes::Bytes {
 /// Deserialize list of op ids.
 pub fn deserialize_op_ids(value: bytes::Bytes) -> K2Result<Vec<OpId>> {
     let op_ids = OpIds::decode(value).map_err(K2Error::other)?;
-    let vec = Vec::from(op_ids);
-    Ok(vec)
+    Ok(op_ids.into())
+}
+
+/// Serialize list of ops for sending over the wire.
+pub fn serialize_ops(value: Vec<MetaOp>) -> bytes::Bytes {
+    let ops = Ops::from(value).encode_to_vec();
+    bytes::Bytes::copy_from_slice(&ops)
```

Comment:
This writes the data, and then copies it again. See: https://github.com/holochain/kitsune2/blob/9cd2ec666f66ebc90d71b445abd612002884e034/crates/api/src/protocol.rs#L15-L24

Reply:
Cool, I'll do that. Question about that protocol code, should that not panic when encoding fails?

Reply:
Indeed. Probably the only way it could error is memory allocation. Up to you : )

Reply:
done https://github.com/holochain/kitsune2/pull/63/commits/4659a4af7162553e50f00b809afa615f6e3010f0
and for op_id serialization https://github.com/holochain/kitsune2/pull/63/commits/3d7fb68e70738c8c1e8a5f53d9621a7bbbad5c76